R/strip_html.r

```diff
@@ -0,0 +1,45 @@
+#' @title strip_html
+#'
+#' @description
+#' Remove all values between "<>", which are typically HTML tags.
+#'
+#' @param dat a data.frame.
+#' @param ignore a character vector containing values to ignore while stripping HTML tags. For instance, if you have <keep me> and <me too> in your column names, add ignore = c("keep me", "me too").
+#'
+#'
+#' @return a data.frame object.
+#'
```

Comment:
Do you want to add an example?  You can put a `# Not run:` ahead of it, then comment the example itself out so it doesn't fail at `check`.  But it might help a user to see how to use it, like `1234567890 %>% fetch_survey_obj() %>% parse_survey() %>% strip_html()`

Reply:
You could also add a sentence about this `strip_html` to the main README home page in the repo, or mention the function in the documentation of `?parse_survey`, so people discover and use this function.